pkg/kubelet/cm/devicemanager/plugin/v1beta1/server.go

```diff
@@ -97,20 +98,44 @@ func (s *server) Start() error {
 		return err
 	}
 
-	ln, err := net.Listen("unix", s.SocketPath())
-	if err != nil {
-		klog.ErrorS(err, "Failed to listen to socket while starting device plugin registry")
-		return err
-	}
-
 	s.wg.Add(1)
-	s.grpc = grpc.NewServer([]grpc.ServerOption{}...)
-
-	api.RegisterRegistrationServer(s.grpc, s)
```

Comment:
in a past we will not return from `Start` before the server was started. Let's keep it as-is